### DISCUSSION TITLE
Batch loopback checkbox

### DISCUSSION THREAD

**Discussion by Evilinside3357** [lllyasviel/stable-diffusion-webui-forge#338] [Ideas]
Is it possible to bring the Batch loopback checkbox for the controlNets? It's a very important option specially for video to video workflows when doing it frame by frame. It can drastically improve consistency between frames.

**Comment by superepic1908** [comment]
I totally agree. Not having that is genuinely my biggest con with forge.

**Comment by DA-Charlie** [comment]
@lllyasviel  will it be added back?
